eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/DTO/SwappedLocationProperties.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Persistence\Legacy\Content\UrlAlias\DTO;
```

Comment:
@alongosz Should we expand DTO to `DataTransfer` or similar?

Reply:
> @alongosz Should we expand DTO to DataTransfer or similar?

@andrerom If you really want... For me DTO acronym is as clear as API or SPI.